pkg/bpf/bpf_linux.go

```diff
@@ -662,6 +662,9 @@ func TestDummyProg(progType ProgType, attachType uint32) error {
 			return errno
 		}
 		return nil
+		// when SELinux enabled on the host, a program running in the container invoke BPF syscall will cause "Permission denied" error.
+	} else if errno == unix.EPERM {
+		log.Warningf("Cilium dosen't have permission to load bpf program. Please disable SELinux and try again.")
```

Comment:
nits for spelling, incorrect formatting print function and unnecessary comment:

```suggestion
	} else if errno == unix.EPERM {
		log.WithError(errno).Warning("Cilium cannot load bpf programs. Security profiles like SELinux may be restricting permissions")
```

Reply:
done